src/libs/migrations/NVPMigration.ts

```diff
@@ -0,0 +1,84 @@
+import after from 'lodash/after';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+const migrations = {
```

Comment:
It might be good to add a quick comment to explain that this is a mapping from the old key name to the new key name. I didn't realize that at first and didn't realize it until digging into the migration.